### PR TITLE
T2-CEPH-83574164-Cephfs subvolumegroup with desired Pool

### PR DESCRIPTION
**Pull request by hkadam134**
Signed-off-by: Hemanth <hyelloji@redhat.com>

# Description
Tier 2 Test - CEPH-83574164-Cephfs subvolumegroup with desired Pool
5.x Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-K0VJ2H/

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by AmarnatReddy — APPROVED** [review]
LTGM

File: conf/pacific/cephfs/tier_2_fs.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+globals:
```
Comment (psathyan) [inline conf/pacific/cephfs/tier_2_fs.yaml:1]: Can we rename this file to `tier-2_cephfs_10-node-cluster.yaml`?

File: suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
```
Comment (psathyan) [inline suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml:1]: Could we rename this file to `tier-2_cephfs_test-volume-management.yaml`?

File: suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      desc: "Configure the Cephfs client system 2"
+      destroy-cluster: false
+      module: test_client.py
+      name: "configure client"
+  - test:
+      name: subvolumegroup creation on desired data pool
+      module: cephfs_volume_management.cephfs_vol_mgmt_subvolgroup_pool_layout.py
+      polarion-id: CEPH-83574164
+      desc: subvolumegroup creation with desired data pool_layout
+      abort-on-fail: true
```
Comment (psathyan) [inline suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml:164]: It's a good practice to end with a newline character.

**Review by psathyan — COMMENTED** [review]
Minor changes. It helps align to the new guidelines.

File: suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml (hunk 2 above)
Comment (hkadam134) [reply suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml:1]: tier-2 folder is not required ??

Comment (psathyan) [reply suites/pacific/cephfs/tier-2/tier-2_cephfs_volume_management.yaml]: Ideally, it is not required but there is not harm in having it a separate folder.

**Review by psathyan — APPROVED** [review]

**Pull request merged**